### PR TITLE
10.0 mig sale exception

### PR DESCRIPTION
**Pull request by mourad-ehm**
[WIP] I proposed an issue to split this module on 2 modules (see #408).

### CODE REVIEW

**Comment by mourad-ehm** [comment]
This PR depends on this one https://github.com/OCA/server-tools/pull/775  .

**Comment by mourad-ehm** [comment]
The commit  b23550c  must be removed before merging this PR.

**Comment by florian-dacosta** [comment]
Travis is still red.
Also, you should update the README file (description is a bit outdated, version number is wrong...)
Also some 'openerp' should be replaced by 'odoo' both in python file for import and in xml files

File: sale_exception/__manifest__.py
Comment (bealdav) [inline sale_exception/__manifest__.py]: Add Camptocamp

**Comment by mourad-ehm** [comment]
Hi @angelmoya could you review this PR. Thanks

File: sale_exception/README.rst
Comment (angelmoya) [inline sale_exception/README.rst]: Best to use short version, check template on mantainer tools

File: sale_exception/__manifest__.py
Comment (angelmoya) [inline sale_exception/__manifest__.py:7]: why 10.0.2.0.0 and not 10.0.1.0.0?

**Review by angelmoya — COMMENTED** [review]
Only few comments, I will check it runbot.

File: sale_exception/__manifest__.py
Comment (mourad-ehm) [reply sale_exception/__manifest__.py:7]: 10.0.2.0.0  because the orgin sale_exception was  splited into 2 modules

**Comment by bealdav** [comment]
:+1:

File: .travis.yml
Comment (sebastienbeau) [inline .travis.yml]: As the main PR is merge can you remove it https://github.com/OCA/server-tools/pull/775

File: oca_dependencies.txt
Comment (sebastienbeau) [inline oca_dependencies.txt]: same here

**Review by sebastienbeau — CHANGES_REQUESTED** [review]

File: oca_dependencies.txt
Comment (mourad-ehm) [reply oca_dependencies.txt]: OK

**Comment by mourad-ehm** [comment]
Hi @sebastienbeau , I maked the change. Could you have a look please ?

**Review by sebastienbeau — APPROVED** [review]

**Comment by sylvainc** [comment]
LGTM (code review) :+1:

**Comment by pedrobaeza** [comment]
Next time, please squash all the "OCA Transbot commit" according to this guide https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests

**Comment by mourad-ehm** [comment]
I w'll do it.

**Pull request merged**
